### PR TITLE
Updated Project Settings article

### PR DESCRIPTION
**Pull request by eshanrnh**
I've added SSL and TLS to the acronyms exception list since these are common terms. I think CDN also can be added but not sure. For now, I've added the full form. Let me know if I should add it to the exception list.

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Project-settings/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ versionTo: 10.0.0
 
 # Project Settings
 
-When working with a Umbraco Cloud project, you can handle a lot of the project configuration directly in the Umbraco Cloud Portal. You can manage team members, database connections, hostnames/domains, security settings, SSL certificates, deployment webhooks, upgrade plan, project usage, advanced settings, rename, or delete the project.
+When working with a Umbraco Cloud project, you can handle a lot of the project configuration directly in the Umbraco Cloud Portal. You can manage team members, database connections, hostnames/domains, security settings, Secure Sockets Layer (SSL) certificates, deployment webhooks, upgrade plan, project usage, advanced settings, rename, or delete the project.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Project-settings/index.md:8]: How about taking the last sentence here, and making a list instead?

**Review by sofietoft — COMMENTED** [review]
Looks good @eshanrnh !

I've made a single suggestion.

File: Umbraco-Cloud/Set-Up/Project-settings/index.md (hunk 1 above)
Comment (eshanrnh) [reply Umbraco-Cloud/Set-Up/Project-settings/index.md]: Done

**Review by sofietoft — APPROVED** [review]
Looks great @eshanrnh ! 🙌

**Pull request merged**